src/pages/about-carbon-for-ibm.com/releases.mdx

```diff
@@ -13,11 +13,11 @@ Learn about our latest updates, our release history, and what the team has plann
 
 Each Carbon for IBM.com release includes package releases that are independently versioned following the industry-standard semantic versioning specification.
 
-The Carbon for IBM.com team also ships package minor releases every other week and when necessary, patch releases between minor versions to address bugs. 
+The Carbon for IBM.com team also ships package minor releases every other week and when necessary, patch releases between minor versions to address bugs.
 
 ### Carbon for IBM.com v2
 
-Carbon for IBM.com focuses on implimenting Carbon Design System v11. Along with simplifying our content authoring and number of pre-constructed components, adding flexibility to our content components, and accessibility improvements. 
+Carbon for IBM.com focuses on implimenting Carbon Design System v11. Along with simplifying our content authoring and number of pre-constructed components, adding flexibility to our content components, and accessibility improvements.
```

Comment:
```suggestion
A primary focus of Carbon for IBM.com is always using the latest version of the Carbon Design System. This allows simplifying our content authoring and number of pre-constructed components, adding flexibility to our content components, and accessibility improvements.
```

I think we can remove the line above this `### Carbon for IBM.com v2` as well. We don't need to refer to the latest version with any number.